### PR TITLE
feat(anni): support musicbrainz

### PR DESCRIPTION
**Pull request by snylonue**

### CODE REVIEW

File: anni/Cargo.toml [hunk 1]
```diff
@@ -36,13 +36,15 @@ anyhow = "1.0"
 cuna = "0.6.3"
 id3 = "1"
 anni-vgmdb = "0.3.0"
+musicbrainz_rs = { git = "https://github.com/snylonue/musicbrainz_rs.git", branch = "release_with_artist_credit"}
```
Comment (Yesterday17) [inline anni/Cargo.toml:39]: publish to crate.io?

**Review by Yesterday17 — COMMENTED** [review]
The other parts LGTM

File: anni/Cargo.toml (hunk 1 above)
Comment (snylonue) [reply anni/Cargo.toml]: This is a patched version. I will open a pr later.

Comment (Yesterday17) [reply anni/Cargo.toml]: If the pull request can not be merged by upstream due to inactivity, just fork this repository to this organization. I've sent an invitation to you.

Comment (snylonue) [reply anni/Cargo.toml]: done

**Review by Yesterday17 — APPROVED** [review]

**Pull request merged**